external-builds/pytorch/build_pytorch_audio.sh

```diff
@@ -0,0 +1,24 @@
+./check_venv.sh
+RES=$?
+if [ ${RES} -ne 0 ]; then
+        echo "Failed to find python virtual-env"
+        echo "Make sure that TheRock has been build first"
+        exit 1
+fi
+
+unset LDFLAGS
+unset CFLAGS
+unset CPPFLAGS
+unset PKG_CONFIG_PATH
+export CMAKE_PREFIX_PATH="$(realpath ../../build/dist/rocm)"
+echo ${CMAKE_PREFIX_PATH}
+export CMAKE_C_COMPILER=${CMAKE_PREFIX_PATH}/bin/hipcc
+export CMAKE_CXX_COMPILER=${CMAKE_PREFIX_PATH}/bin/hipcc
+export HIP_DEVICE_LIB_PATH=${CMAKE_PREFIX_PATH}/lib/llvm/amdgcn/bitcode
+
+cd pytorch_audio
+ROCM_PATH=${CMAKE_PREFIX_PATH} USE_ROCM=1 USE_CUDA=0 USE_FFMPEG=1 USE_OPENMP=1 BUILD_SOX=0 CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} BUILD_VERSION="2.6.0" BUILD_NUMBER=1 python setup.py bdist_wheel
+if [ -e dist/torchaudio-2.6.0-cp312-cp312-linux_x86_64.whl ]; then
```

Comment:
This shouldn't be conditional. Things shouldn't fail silently, and shouldn't be hardcoded like this.

Prefer something like: `pip install -f dist torchaudio --no-index --no-deps` then `pip install torchaudio` to get the deps. I'd not at a terminal and would need to find the recipe I usually use for this but it is something like that.

Shouldn't hardcode version, python version, or platform.